util/pyclient/bft_client.py

```diff
@@ -538,9 +540,11 @@ async def _send_data(self, data, dest_replica):
             # dest_replica is not connected (crushed? isolated?)
             self.establish_ssl_stream_parklot[dest_addr].unpark()
             return
-        # first 4 bytes include the data header (message size), then comes the data
+        # first 4 bytes include the data header (message size), then comes the endpoint and data
         data_len = len(data)
-        out_buff = bytearray(data_len.to_bytes(self.MSG_HEADER_SIZE, "big"))
+        out_buff = bytearray(data_len.to_bytes(self.MSG_LEN_SIZE, "big"))
+        end_point_num = 0xffff # some random endpoint
```

Comment:
Why random? We need the ability to set this value from outside (by tests). Please give it a default value 0xffffffff, like in the c++ code, and add it as a parameter to the send functions.

Reply:
updated.